table/block_based_table_reader.h

```diff
@@ -429,7 +431,7 @@ struct BlockBasedTable::Rep {
 
   const ImmutableCFOptions& ioptions;
   const EnvOptions& env_options;
-  const BlockBasedTableOptions& table_options;
```

Comment:
Can it be fixed by removing `&` here?

Reply:
Yes, it can be fixed by removing `&`. However, it will increase memory usage and incur memory copies.

Reply:
Sounds good. I'll close this PR and open new one.